disperser/common/v2/blobstore/s3_blob_store.go

```diff
@@ -0,0 +1,42 @@
+package blobstore
+
+import (
+	"context"
+
+	"github.com/Layr-Labs/eigenda/common/aws/s3"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+)
+
+type BlobStore struct {
+	bucketName string
+	s3Client   s3.Client
+	logger     logging.Logger
+}
+
+func NewBlobStore(s3BucketName string, s3Client s3.Client, logger logging.Logger) *BlobStore {
+	return &BlobStore{
+		bucketName: s3BucketName,
+		s3Client:   s3Client,
+		logger:     logger,
+	}
+}
+
+// StoreBlob adds a blob to the blob store
+func (b *BlobStore) StoreBlob(ctx context.Context, blobKey string, data []byte) error {
+	err := b.s3Client.UploadObject(ctx, b.bucketName, blobKey, data)
+	if err != nil {
+		b.logger.Errorf("failed to upload blob in bucket %s: %v", b.bucketName, err)
+		return err
+	}
+	return nil
+}
+
+// GetBlob retrieves a blob from the blob store
+func (b *BlobStore) GetBlob(ctx context.Context, blobKey string) ([]byte, error) {
+	data, err := b.s3Client.DownloadObject(ctx, b.bucketName, blobKey)
+	if err != nil {
+		b.logger.Errorf("failed to download blob from bucket %s: %v", b.bucketName, err)
+		return nil, err
+	}
+	return data, nil
+}
```

Comment:
When we eventually merge the [fragmented uploading utility](https://github.com/Layr-Labs/eigenda/pull/833), we will want to revisit this code. A prerequisite to using fragmented uploads will be to store extra metadata for each blob (i.e. fragment size + blob size needed to download without round trips).

No need to wait to merge this PR. The fact that blobs are fragmeneted in S3 should be invisible to things utilizing this API. As long as we don't deploy this code to production environments, a switchover to the fragmented strategy won't be that tricky.

Reply:
Not sure if we want the fragmented uploading in the blob store. This is not the relay store that stores chunks, but the one that stores whole blobs. I think this simple wrapper can serve the blob store in the long run too?

Reply:
I'm ok waiting to have the discussion for how we want to store blobs. It's mostly a question as to whether we want to try and improve latency for really big blobs. Since we don't really support super big blobs currently, it might be something we can avoid dealing with for the MVP.